src/main/java/com/jmb/springfactory/model/bo/QueryTaskObject.java

```diff
@@ -14,6 +14,7 @@
 
     private Integer scheduleId;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "scheduleId" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)